src/index.html

```diff
@@ -7,20 +7,21 @@
   <title>Document</title>
 </head>
 <body>
-  <h1>Leaderboard</h1>
+  <h1>Leader<span>board</span></h1>
   <section class="leaderboard flex">
     <div class="score">
       <div class="scorediv">
         <div class="recent flex">
-          <span>Recent Scores</span>
+          <span>Recent <span class="color">Scores</span></span>
           <button class="refresh">Refresh</button>
         </div>
         <ul class="scores">
+          <h2>Refresh Leaderboard</h2>
         </ul>
       </div>
     </div>
     <form class="form flex" >
-      <span>Add Your Score</span>
+      <span>Add Your <span class="color">Score</span></span>
       <label for="Name">
         <input class="leader user" name="Name" type="text" placeholder="your name" maxlength="30" required>
       </label>
```

Comment:
- [ ] Kindly align `Recent scores` and `Add new scores` so they look similar to the given wireframe

Yours:
![image](https://user-images.githubusercontent.com/62705947/195812874-1a2d52bd-ade7-4b31-a30f-8f8ccd37b144.png)

Wireframe:
![image](https://user-images.githubusercontent.com/62705947/195812741-fd4a81e4-8b4e-41b0-8eee-8042c162d603.png)

Reply:
Thank you for the review I'll do well to put the changes for the next reviewer.👌👍